network.go

```diff
@@ -146,7 +155,8 @@ func startMDNS() error {
 	}
 
 	mDNSConn, err = mdns.Server(ipv4.NewPacketConn(l4), ipv6.NewPacketConn(l6), &mdns.Config{
-		LocalNames: []string{"jetkvm.local"}, //TODO: make it configurable
+		LocalNames: []string{config.NameConfig.DNS},
+		//LocalNames: []string{"jetkvm.local"}, //TODO: make it configurable
```

Comment:
```suggestion
```

Reply:
The `.local` should probably be kept/appended here as it is the proper TLD reserved for local DNS address (and specifically mDNS)
https://en.wikipedia.org/wiki/.local